ee/frontend/mobile-replay/transformers.ts

```diff
@@ -121,6 +124,59 @@ function makeImageElement(wireframe: wireframeImage, children: serializedNodeWit
     }
 }
 
+function inputAttributes<T extends wireframeInputComponent>(wireframe: T): attributes {
+    const attributes = {
+        style: makeStylesString(wireframe),
+        type: wireframe.inputType,
+        ...(wireframe.disabled ? { disabled: wireframe.disabled } : {}),
```

Comment:
if `disabled` not present, we should always fallback to `disabled=false`, either here or on the SDK

Reply:
rrweb represents these as true or absent. so here if disabled/checked/selected then we set them, otherwise we leave them out (which is equivalent to false)

makes this code a bit convoluted but meets rrweb's needs